Binary files /opt/origin/portage/sci-libs/rocSPARSE-4.3.0/work/rocSPARSE-rocm-4.3.0/deps/convert and /opt/build/portage/sci-libs/rocSPARSE-4.3.0/work/rocSPARSE-rocm-4.3.0/deps/convert differ
diff --color -upr /opt/origin/portage/sci-libs/rocSPARSE-4.3.0/work/rocSPARSE-rocm-4.3.0/library/src/include/common.h /opt/build/portage/sci-libs/rocSPARSE-4.3.0/work/rocSPARSE-rocm-4.3.0/library/src/include/common.h
--- /opt/origin/portage/sci-libs/rocSPARSE-4.3.0/work/rocSPARSE-rocm-4.3.0/library/src/include/common.h	2021-05-27 23:04:12.000000000 +0800
+++ /opt/build/portage/sci-libs/rocSPARSE-4.3.0/work/rocSPARSE-rocm-4.3.0/library/src/include/common.h	2021-09-30 22:32:09.203071933 +0800
@@ -171,7 +171,7 @@ __device__ __forceinline__ void rocspars
     if(BLOCKSIZE >   1) { if(i <   1 && i +   1 < BLOCKSIZE) { data[i] = min(data[i], data[i +   1]); } __syncthreads(); }
 }
 
-#ifndef __gfx1030__
+#ifndef __gfx1031__
 // DPP-based wavefront reduction maximum
 template <unsigned int WFSIZE>
 __device__ __forceinline__ void rocsparse_wfreduce_max(int* maximum)
